production/aggregator/README.md

```diff
@@ -0,0 +1,13 @@
+### Create Sealed secret
```

Comment:
I would try to find a better place to place the documentation and not duplicating it.
Alternatives: 
- Put the documentation only in Production
- Put the documentation only in Staging
- Create a directory only for documentation independent of the environment